scripts/rvps-extraction/GetRvps.sh

```diff
@@ -3,60 +3,75 @@
 
 function install_packages() {
     echo "***Installing necessary packages for RVPS values extraction ***"
-    dnf install -y python3 python3-cryptography kmod 
+    dnf install -y python3 python3-cryptography kmod
     echo "***Installation Finished ***"
 }
 
 # Function to mount the image and extract se.img
 function mount_and_extract_image() {
     local img_path=$1
-    
+
     # Cleanup any previous files and directories
     rm -rf se.img /mnt/myvm
     mkdir /mnt/myvm
-    
+
     # Load nbd module and mount the image
     modprobe nbd
     if [ $? -ne 0 ]; then
         echo "Error: Failed to load nbd module."
         exit 1
     fi
 
-    qemu-nbd -c /dev/nbd3 $img_path
+
+    #Check the available nbd and use it
+    nbd_available=`lsblk | grep nbd | grep 0B | awk '{print $1}' | head -1`
+    echo $nbd_available "looks available. Starting Mounting the image with it.."
```

Comment:
maybe worth checking the string exist (and exit if not?) as it may print the msg also when it's nor available

Reply:
Good catch @snir911 ! It will indeed print a `  looks available. Starting Mounting the image with it..` message and `qemu-nbd -c /dev/ $img_path` will then fail right away with `qemu-nbd: Failed to open /dev: Is a directory`. Testing the string isn't empty will allow to come up with a more meaningful error message.

Please process this comment @gauravkuredhat.

Reply:
Changed @snir911 @gkurz . thanks